src/lib/providers/adobeanalytics/adobeanalytics.ts

```diff
@@ -14,18 +14,27 @@ export class Angulartics2AdobeAnalytics {
   ) {
     this.angulartics2.pageTrack
       .pipe(this.angulartics2.filterDeveloperMode())
-      .subscribe((x) => this.pageTrack(x.path));
+      .subscribe((x) => this.pageTrack(x.path, x.properties));
     this.angulartics2.eventTrack
       .pipe(this.angulartics2.filterDeveloperMode())
       .subscribe((x) => this.eventTrack(x.action, x.properties));
     this.angulartics2.setUserProperties
       .subscribe((x) => this.setUserProperties(x));
   }
 
-  pageTrack(path: string) {
+  pageTrack(path: string, properties: any) {
```

Comment:
what is the return type for both `pageTrack` and `eventTrack`? The original provider and all the other providers don't show any return types (neither `void` nor `any`).

Reply:
i think its just an object of some kind

Reply:
Can we send here new property with category and label ? If yes then please guide me how we will send here .
Currently i send like below way but it not workout.It is not tracked in ga report.
this.angulartics2.eventTrack.next({
    action: action,
    properties: {
    category: category,
     label: label,
     search_query:'xyz'
}});